MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpressHci.c

```diff
@@ -1126,9 +1126,7 @@ NvmeUnregisterShutdownNotification (
 {
   EFI_STATUS                      Status;
   EFI_RESET_NOTIFICATION_PROTOCOL *ResetNotify;
-  //MS_CHANGE - BEGIN
-  ReportStatusCode((EFI_ERROR_MAJOR | EFI_ERROR_CODE), (EFI_IO_BUS_SCSI | EFI_IOB_EC_INTERFACE_ERROR));

```

Comment:
Would these report status code removals be good candidates for instead pivoting to emitting telemetry here?

Reply:
no.  telemetry is not in basecore yet